test/core/test_alloca.out

```diff
@@ -1 +1 @@
-z:1*
\ No newline at end of file
+
```

Comment:
Do we need this file if it's empty?

Reply:
There is no useful output from this test.. so I guess I can convert the type of test to one that doesn't expect an output file and delete this.